operations/mimir-mixin/dashboards/queries.libsonnet

```diff
@@ -235,23 +236,27 @@ local filename = 'mimir-queries.json';
           |||
         ) +
         $.queryPanel(
-          [
+          local ncSumRate = utils.ncHistogramSumBy(utils.ncHistogramCountRate($.queries.ingester.requestsPerSecondMetric, $.queries.ingester.readRequestsPerSecondSelector));
+          local scSuccessful =
             |||
               (
                 sum(rate(cortex_ingest_storage_strong_consistency_requests_total{%s}[$__rate_interval]))
                 -
                 sum(rate(cortex_ingest_storage_strong_consistency_failures_total{%s}[$__rate_interval]))
               )
-              /
-              sum(rate(cortex_request_duration_seconds_count{%s,route=~"%s"}[$__rate_interval]))
-            ||| % [$.jobMatcher($._config.job_names.ingester), $.jobMatcher($._config.job_names.ingester), $.jobMatcher($._config.job_names.ingester), $._config.ingester_read_path_routes_regex],
+            ||| % [$.jobMatcher($._config.job_names.ingester), $.jobMatcher($._config.job_names.ingester)];
+          local scFailed =
             |||
               sum(rate(cortex_ingest_storage_strong_consistency_failures_total{%s}[$__rate_interval]))
-              /
-              sum(rate(cortex_request_duration_seconds_count{%s,route=~"%s"}[$__rate_interval]))
-            ||| % [$.jobMatcher($._config.job_names.ingester), $.jobMatcher($._config.job_names.ingester), $._config.ingester_read_path_routes_regex],
+            ||| % [$.jobMatcher($._config.job_names.ingester)];
+          local scRate(sc, rate) = std.join(' / ', [sc, rate]);
```

Comment:
Technically works, but mathematically you'd want to apply util.showClassicHistogramQuery last as it applies to the whole expression. Anyway, that would probably make it more complicated.